tests/internal/scrollable-container-test.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
+<html lang=en>
+  <meta charset="utf-8">
+  <title>Spatnav sanity check</title>
+  <script src="../../polyfill/spatial-navigation-polyfill.js"></script>
+  </script>
+  <link rel="stylesheet" href="../../demo/sample/spatnav-style.css">
+  <link rel="stylesheet" href="test.css">
+  <script src="test.js"></script>
+  <style>
+  #d {
+    width: 100px;
+    height: 100px;
+    overflow: scroll;
+  }
+  </style>
+  <body onload="onload()">
+    <div id="c1" class="container" tabindex="0" style="width:600px; height:100px; overflow-y: scroll;">
+      <button id="c1b1" class="box" style="top: 50px; left: 100px;"></button>
```

Comment:
Same as above. (top: 50px; left: 100px)

Reply:
This review is a bit confusing because this is about 'scrollable container' but this PR is about `spatial-navigation-action: auto`.
And there had been several changes after merging this PR. I'll fix this part from another commit or PR.